### PR TITLE
Add known limitations to README

### PR DESCRIPTION
**Pull request by dmytrotur**
## Purpose
Add known limitations about tenant-selected timezone and locale support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -396,6 +396,12 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+## Known limitations
```
Comment (marcjohnson-kint) [inline README.md]: There is already a [known limitations](https://github.com/folio-org/mod-circulation/blob/master/README.md#known-limitations) section in the readme, please can this be combined together with that

File: README.md [hunk 2]
```diff
@@ -396,6 +396,12 @@ Loans include information from the item, including locations, holdingsRecordId a
...
+
+* Tenant-selected timezone is supported only for check-out and renew
+* Patron notices are implemented for check-in and check-out
```
Comment (marcjohnson-kint) [inline README.md]:
```suggestion
* Patron notices are only implemented for check-in and check-out
```

File: README.md [hunk 3]
```diff
@@ -396,6 +396,12 @@ Loans include information from the item, including locations, holdingsRecordId a
 
 Requests include information from the item, including holdingsRecordId and instanceId.
 
+## Known limitations
+
+* Tenant-selected timezone is supported only for check-out and renew
```
Comment (marcjohnson-kint) [inline README.md]:
```suggestion
* Tenant-selected timezone is supported only for check-out and renewal (and not for overridden renewals or request creation) 
```

File: README.md (hunk 1 above)
Comment (dmytrotur) [reply README.md]: Sure, I'll move it here.

**Pull request merged**